blocks/simple-list-block/features/simple-list/default.test.jsx

```diff
@@ -156,6 +156,12 @@ describe("Simple list", () => {
 		expect(wrapper.find("StoryItem").length).toBe(2);
 	});
 
+	it("should render content only for the arcSite", () => {
```

Comment:
@JackHowa how do you feel about this switch? putting this test before the "should not render items when no data provided" seemed to resolve a mock issue. I'm not that great at Jest so happy to take feedback

Reply:
the order shouldn't matter with mocking. but we can pair -- or you and Joe -- on mocking and jest. there's a few other examples of mocking the content and isServerSide in the numbered list block if you're looking for oher examples

Reply:
updated!